src/SimpleXmlBuilder.php

```diff
@@ -28,82 +28,44 @@ class SimpleXmlBuilder extends SimpleXMLElement
     private static $dangerousCharacters = '\'"><&';
 
     /**
-     * Creates an xml document from an array.
-     * Array keys @attributes can have associative arrays which will be converted
-     * to attributes for the parent.
-     * Array keys @namespace to define a namespace for a node.
+     * Wrapper for addAttribute to allow for an array of attributes.
      *
-     * @param array $document
-     * @param null|SimpleXmlBuilder $xmlDocument a SimpleXmlBuilder document to
-     *     append to.
-     * @return null|SimpleXmlBuilder Will return null if the $document was empty.
+     * @param array $attributes
+     * @return void
      */
-    public static function createXML(array $document, SimpleXmlBuilder &$xmlDocument = null)
+    public function addAttributes(array $attributes)
     {
-        foreach ($document as $element => $values) {
-            $namespace = null;
-            $attributes = '';
+        foreach ($attributes as $name => $value) {
+            if (!is_scalar($name)) {
```

Comment:
Yes thats true, i'll revert it back to `is_string`. The more extensive checking will be done by SimpleXMLElement